cmd/buffer-api/main.go

```diff
@@ -72,7 +72,7 @@ func start(host, port string, debug, debugHTTP bool, stdLogger *stdLog.Logger, e
 	defer cancelFn()
 
 	// Create logger.
-	logger := log.NewAPILogger(stdLogger, "", debug)
+	logger := log.NewAPILogger(stdLogger, debug).AddPrefix("[bufferApi]")
```

Comment:
A este som odstranil jeden technicky dlh, ktory sme tu mali.

V kode sme mali dependencies:
- `Logger() log.Logger`
- `PrefixLogger() log.PrefixLogger`

Malo to riesit pridavanie prefixu (kontextu) do log messages.
Nahradil som to za `log.Logger.AddPrefix`.

V Q1 sa ma presjt na DataDog Logs, tak potom sa na to logovanie pozrieme lepsie.
Namiesto message budeme logovat asi JSON, a `context` budu nejake fields v tom JSON.
Takze tu je len jeden krok z toho co bude nasledovat neskor.

Reply:
Mame na to E2E test `endpoint-not-found`, ktory kontroluje aj logs v stderr a otestoval som to aj rucne:
![image](https://user-images.githubusercontent.com/19371734/205656900-dfb3d4c9-c550-4e20-8a47-035ef78c7542.png)

-------

Teda nic sa funkcne nezmenilo.

Reply:
BTW mozno by sme mohli dat za `prefix`, pred `INFO` (level) medzeru, ale radsej som to nemenil, nech tu nie je vela zmien.